board/grinn/chiliboard-lcd-res/genimage.cfg

```diff
@@ -0,0 +1,31 @@
+# Minimal microSD card image for Grinn's chiliboard with relay board
```

Comment:
This is LCD, not relay board. Please also verify other commits. Maybe just remove "with *** board" from that comment? It does not look necessary here.